### ISSUE TITLE
'/home/$USER/.vsdbg/getvsdbg.sh' is missing after fresh install

### ISSUE THREAD

**Issue by tehho** [Microsoft/vscode-docker#897]
TLDR:
Error: EACCES: permission denied, open '/home/$USER/.vsdbg/getvsdbg.sh' but '/home/$USER/.vsdbg/getvsdbg.sh' is missing.
Fix install plox :D

Repro steps:

1. Fresh install of pop_os
2. Install vs code
3. dotnet new mcv
4. standard dockerfile
5. standard docker-coreclr buildtask
6. Build goes thru but run does not.

Error Message:
Acquiring the latest .NET Core debugger...
Unable to acquire the .NET Core debugger.
Error: EACCES: permission denied, open '/home/$USER/.vsdbg/getvsdbg.sh'

Action: debugCoreClr
Error type: EACCES
Error Message: EACCES: permission denied, open '/home/tehho/.vsdbg/getvsdbg.sh'

Version: 0.6.1
OS: linux
VS Code Version: 1.33.1

**Comment by philliphoff** [comment]
@tehho Thanks for the report; I'll take a look at the extension in this OS (which I've not tried before).

**Comment by philliphoff** [comment]
The issue is due to the container being started with a bind mount to the debugger folder, before the debugger folder has been created (or debugger bits acquired).  We have to start the container before debugger acquisition in order to know which debugger to acquire.  However, if the mounted folder does not exist when the container runs, Docker will create the folder itself, and the folder will be owned by Docker.  However, Docker may run with different permissions than VS Code (i.e. the user), and so VS Code may not be able to write into the folder thereafter.

The fix is to ensure the debugger folder has been created prior to the container being run (irrespective of whether debugger bits have been acquired).

**Comment by philliphoff** [comment]
@tehho A workaround should be to remove the `~/.vsdbg` folder and manually re-create the folder with the same permissions as those used by VS Code (i.e. the current user).
